code/game/machinery/newscaster.dm

```diff
@@ -42,8 +43,10 @@
 	is_admin_channel = 0
 	total_view_count = 0
 
-/datum/feed_channel/proc/announce_news()
-	return "Breaking news from [channel_name]!"
+/datum/feed_channel/proc/announce_news(title="")
+	if(title)
+		return "Breaking news from [channel_name]: [title]"
```

Comment:
There should probably be an exclamation mark here, for more !!excitement!!

Reply:
Oh! Yeah, I took that exclamation off, cause an exclamation mark is already applied here:
https://github.com/ParadiseSS13/Paradise/blob/1877df65aa61016b928582d4e26d34310117cc22/code/game/machinery/newscaster.dm#L827

Reply:
oh, 👍 then